rules/app.bzl

```diff
@@ -56,7 +57,16 @@ def ios_application(name, apple_library = apple_library, **kwargs):
         apple_library: The macro used to package sources into a library.
         **kwargs: Arguments passed to the apple_library and ios_application rules as appropriate.
     """
-    infoplists = write_info_plists_if_needed(name = name, plists = kwargs.pop("infoplists", []))
+
+    infoplists_by_build_setting = kwargs.pop("infoplists_by_build_setting", {})
```

Comment:
Could we update the app_doc.md for this argument? It's specific to ios_application, and not supported in rules_apple/ios_application.

Reply:
Will do 👍